streams/src/main/java/org/apache/kafka/streams/kstream/WindowedSerdes.java

```diff
@@ -30,6 +30,13 @@ public TimeWindowedSerde() {
         public TimeWindowedSerde(final Serde<T> inner) {
             super(new TimeWindowedSerializer<>(inner.serializer()), new TimeWindowedDeserializer<>(inner.deserializer()));
         }
+
+        public TimeWindowedSerde(final Serde<T> inner, final long windowSize) {
```

Comment:
Is this mentioned in the KIP? It's a public API change.

Reply:
Ah, no. When I did this before, I did it differently to keep it private. I thought this was a better way, but overlooked the public-ness of it.

I'll go back to private mode.